packages/integration-tests/src/api/me.ts

```diff
@@ -1,35 +0,0 @@
-import type { ArbitraryObject, UserInfo } from '@logto/schemas';
```

Comment:
this file removed before adding the new `/me` APIs. will bring it back in the following PRs